src/rc/seedrng.c

```diff
@@ -392,10 +433,27 @@ int main(int argc _unused, char *argv[] _unused)
 	struct timespec realtime = { 0 }, boottime = { 0 };
 	struct blake2s_state hash;
 
+	atexit(cleanup);
+	applet = basename_c(argv[0]);
+	if (geteuid())
+		eerrorx("%s: superuser access is required", applet);
 	umask(0077);
-	if (getuid())
-		eerrorx("This rc helper program requires root");
 
+	while ((opt = getopt_long(argc, argv, getoptstring,
+		    longopts, (int *) 0)) != -1)
+	{
+		switch (opt) {
+			case LONGOPT_LOCK_FILE:
+			if (!lock_file)
+				lock_file = xstrdup(optarg);
+			break;
+		case LONGOPT_SEED_DIR:
+			if (!seed_dir)
+				seed_dir = xstrdup(optarg);
+			break;
+		case_RC_COMMON_GETOPT
```

Comment:
This looks like a typo. Should be

````c
case RC_COMMON_GETOPT:
````

Reply:
With... presumably something in the body. Or maybe this line just needs to be deleted. Dunno.

Reply:
It's a macro: https://github.com/OpenRC/openrc/blob/301161a7a4bda27c7ebd54c537dfec0cbc926365/src/rc/_usage.h#L39-L45